<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>527</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="clearPushButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="vertexModeRad">
         <property name="text">
          <string>Vertex Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="translateModeRad">
         <property name="text">
          <string>Translate Mode</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Current Mode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="currModeLabel"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="MyOpenGLWidget" name="openGLWidget">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Current Mode: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="currAxisLabel"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="xAxisRad">
       <property name="text">
        <string>X-Axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="yAxisRad">
       <property name="text">
        <string>Y-Axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="zAxisRad">
       <property name="text">
        <string>Z-Axis</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>myopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
